### PR TITLE
Update NUnit to NUnit3

### PR DESCRIPTION
**Pull request by simonhdickson**
This updates from NUnit2 to NUnit3.

This fixes the build from the command line on OSX, it will also enable me to resurrect my Homebrew PR. :)

### CODE REVIEW

**Comment by forki** [comment]
Interesting. Will check out tomorrow

File: tests/Paket.Tests/PackageProcessSpecs.fs [hunk 1]
```diff
@@ -11,7 +11,7 @@ let ``Loading assembly metadata works``() =
     let workingDir = Path.GetFullPath(".")
     
     let fileName =
-        Path.Combine(workingDir, "..", "..", "Paket.Tests.fsproj")
+        Path.Combine(workingDir, "Paket.Tests.fsproj")
```
Comment (forki) [inline tests/Paket.Tests/PackageProcessSpecs.fs]: What's wrong here?

Comment (forki) [inline tests/Paket.Tests/PackageProcessSpecs.fs]: image

Comment (simonhdickson) [inline tests/Paket.Tests/PackageProcessSpecs.fs]: I'm not quite sure, it just fails under NUnit3 without that as it looks in the wrong folder.

Comment (forki) [inline tests/Paket.Tests/PackageProcessSpecs.fs]: let me try somthing

Comment (simonhdickson) [inline tests/Paket.Tests/PackageProcessSpecs.fs]: Hmm interesting, I don't get that on Mac or Windows, maybe there's something off with my setup.

**Comment by forki** [comment]
merge

Great work!

**Pull request merged**